Add logs support (#7791)
DataDog/integrations-core: kyototycoon/README.md

2. [Restart the Agent][4].

##### Log collection

> 

Author: ```suggestion
```

2. [Restart the Agent][4].

##### Log collection

> _Available for Agent versions >6.0_

Reviewer: ```suggestion
```
this is outdated

1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:

    ```yaml
    logs_enabled: true
    ```

2. Add this configuration block to your `kyototycoon.d/conf.yaml` file to start collecting Kyoto Tycoon logs:

    ```yaml
    logs:
      - type: file
        path: /var/data/ktserver.log
        source: kyototycoon

>         service: "<SERVICE_NAME>"

Reviewer 2: How come `service` isn't kyototycoon?

Author: Thanks, now supersuded as per Christine's comment - the `service` option here was outdated.

>         service: "<SERVICE_NAME>"

Reviewer: ```suggestion
```

dropping in favor of `service` config option in init_config/instance level

Author: I see! Are those propagated to log collection too?

Reviewer: Yes, hence the change for `service` to become `optional` in config now

    ```

>     Change the `path` and `service` parameter values based on your environment. See the [sample kyototycoon.d/conf.yaml][3] for all available configuration options.

Reviewer: ```suggestion
    Change the `path` parameter value based on your environment. See the [sample kyototycoon.d/conf.yaml][3] for all available configuration options.
```
